prob44/prob44.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <script type="text/javascript">
+      const calculate=(array)=> {
+        var stats=[];
```

Comment:
Let's use ES6 `const` and `lets`

Reply:
for whatever reason i cant use ES6 on the laptop

Reply:
😱 That's a problem!

Reply:
i know

Reply:
whenever I do it gives me a "Uncaught SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode" error